membership/models.py

```diff
@@ -19,7 +19,7 @@
                         'from': 'test@test.com'}
 
 
-def get_anonymous_user_instance():
+def get_anonymous_user_instance(CustomUser):
```

Comment:
I had issues with the name of the variable chosen `CustomUser`, but django-guardian suggests that this is the way to go.

https://media.readthedocs.org/pdf/django-guardian/v1.3.1/django-guardian.pdf 
Page: 17-18 -> Section 5.5: CustomUser